### ISSUE TITLE
Use a distinctive app icon

### ISSUE THREAD

**Issue by aadilayub** [fraction/oasis-desktop#7]
It would be cool if oasis had its own app icon, instead of using the same hermit crab icon that Patchwork and Patchbay use (I can imagine this being a source of confusion for people who have all 3 apps installed).

Maybe we could just use the 🏝️ emoji?

**Comment by christianbundy** [comment]
Good idea! The Oasis SVG is available here: https://raw.githubusercontent.com/fraction/oasis/master/src/assets/favicon.svg

**Comment by aadilayub** [comment]
@christianbundy would you happen to know how to generate the corresponding .ico and .icns files from this svg?

**Comment by christianbundy** [comment]
Not off the top of my head, I'd have to look it up. My guess is that imagemagick lets us do this.

**Comment by aadilayub** [comment]
I figured it out! Made a PR https://github.com/fraction/oasis-desktop/pull/10
